src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(MAX_NUMBER) + 1;
+        return new Ball(randomColor, randomNumber);
+    }
```

Comment:
Consider removing the static modifier from the `getRandomBall` method to comply with the best practices mentioned in the checklist. Static methods are generally discouraged in this context.